packages/circuit-ui/components/Display/Display.tsx

```diff
@@ -22,6 +22,13 @@ import { deprecate } from '../../util/logger.js';
 import classes from './Display.module.css';
 
 export interface DisplayProps extends HTMLAttributes<HTMLHeadingElement> {
+  /**
+   * Choose from three font weights. Default: `regular`.
+   *
+   * Use the `as` prop to render the component as the `strong` HTML element
+   * if appropriate.
```

Comment:
I'd remove this guidance since the Display component should be rendered as a heading element even when bold.